toolchains/common/packages/libsdl1.2/build.sh

```diff
@@ -10,7 +10,7 @@ do_make_bdir
 
 do_http_fetch SDL "https://github.com/libsdl-org/SDL-1.2/archive/${SDL_VERSION}.tar.gz" 'tar xzf'
```

Comment:
If I could humbly recommend something, please use https://github.com/mikrosk/SDL-1.2 instead. It's full of Atari/FreeMiNT-specific fixes and improvements. I regularly merge changes from the upstream there.

Reply:
I'm OK with doing that, although it might be worth submitting those changes upstream at some point.

Reply:
> I'm OK with doing that, although it might be worth submitting those changes upstream at some point.

FYI, this just has been done. No need for my private branch anymore.